docs/source/edge_detection.rst

```diff
@@ -0,0 +1,245 @@
+.. _edge_detection:
+
+Edge Detection
+==============
+
+With this application, users can create lines along edges (lineaments)
+from gridded data in a semi-automated fashion. The application uses
+machine vision algorithms from the
+`Scikit-Image <https://scikit-image.org/>`__ open-source package. Edges are exported to `Geoscience ANALYST <https://mirageoscience.com/mining-industry-software/geoscience-analyst/>`__
+for viewing and editing.
+
+.. figure:: ./images/edge_detection/edge_detection_landing.png
+            :align: center
+            :width: 100%
+
+New user? Visit the `Getting Started <getting_started>`_ page.
```

Comment:
Link is broken

Reply:
These won't work until we are fully build